src/constants/userRoles.js

```diff
@@ -1,4 +1,6 @@
 export const userRoles = {
-    GUEST: 'Guest',
-    USER: 'User'    // when adding a role, add it to authReducer in the set access case
-}
+    GUEST: "Guest",
+    USER: "User", // when adding a role, add it to authReducer in the set access case
+    RECIPIENT: "Recipient",
```

Comment:
Is it better to separate this two roles (Recipient and Sender) in offerRoles constant?